packages/module/package.json

```diff
@@ -1,68 +1,54 @@
 {
-  "name": "@ausuliv/react-log-viewer",
-  "version": "4.84.2",
+  "name": "@patternfly/react-log-viewer",
+  "version": "1.0.0",
```

Comment:
Should we reset the version here to 1.0.0? Or should we increment from the current log viewer release version number?

Reply:
we should upgrade the version